pages/api/items/[id].js

```diff
@@ -0,0 +1,35 @@
+import { getItem, updateItem } from "@/helpers/db";
+
+export default async function handler(request, response) {
+  switch (request.method) {
+    case "GET": {
+      const item = await getItem(request.query.id);
+      if (!item) {
+        response.status(404).json({
+          message: `Item ${request.query.id} was not found.`,
+        });
```

Comment:
This code does not terminate the function and another response will be attempted to be send a few lines below directly after execution of this code.